src/main/java/org/tugraz/sysds/runtime/data/DenseBlockFP64.java

```diff
@@ -187,4 +193,9 @@ public double get(int r, int c) {
 	public double get(int[] ix) {
 		return _data[pos(ix)];
 	}
+
+	@Override
+	public String getString(int[] ix) {
```

Comment:
Apart from the String modifications, I guess we need to slightly generalize this implementation because otherwise it would not work for combinations of different value types (e.g., on set dense block). However, please be cautious as this double dense block is already in active use, so we need to make sure there are no performance regressions.